packages/env/lib/cli.js

```diff
@@ -109,6 +109,28 @@ module.exports = function cli() {
 		},
 		withSpinner( env.clean )
 	);
+	yargs.command(
+		'log',
+		'displays PHP and Docker logs for given WordPress environment.',
+		( args ) => {
+			args.positional( 'environment', {
+				type: 'string',
+				describe: 'Which environment to display the logs from.',
+				choices: [ 'development', 'tests' ],
+				default: 'development',
+			} );
+			args.positional( 'watch', {
```

Comment:
I think this should be an `option`, not a `positional`.

Reply:
done!